### PR TITLE
divided program into classes and added comment lines(not very good)

### PR DESCRIPTION
**Pull request by radudin11**
please review these changes

### CODE REVIEW

File: Card.cs [hunk 1]
```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace cardGameSix
+{
+    public class Card
```
Comment (radudin11) [inline Card.cs:7]: gbjh vjhgvhfgc

File: CardSign.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace cardGameSix
+{
+    public enum Sign : int { HEARTS = 1, DIAMONDS = 2, SPADES = 3, CLUBS = 4 };
```
Comment (rodede) [inline CardSign.cs]: I think the name `Sign` should be the same as the file name a`CardSign` - use refactor (rename field) https://docs.microsoft.com/en-us/visualstudio/ide/reference/rename?view=vs-2019 here and below

```suggestion
    public enum CardSign : int { HEARTS = 1, DIAMONDS = 2, SPADES = 3, CLUBS = 4 };
```

**Review by radudin11 — COMMENTED** [review]
please check if it's all good for merging with the main branch

File: Deck.cs [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        {
+            foreach (Card card in deck)
+            {
+                Console.Write(card + " ");
+            }
+        }
+        public Deck()
+        {
+            deck = new Stack<Card>();
+            foreach (Value v in Card.values)
```
Comment (rodede) [inline Deck.cs]: this was renamed - the project is not running anymore!

**Comment by rodede** [comment]
looks good!

**Review by rodede — APPROVED** [review]
LGTM

**Pull request merged**